gallery/src/components/NxBinaryDonutChart/NxBinaryDonutChartPage.tsx

```diff
@@ -27,6 +27,18 @@ const NxBinaryDonutChartPage = () =>
       <NxP>
         <NxCode>NxBinaryDonutChart</NxCode> represents a binary donut chart.
       </NxP>
+      <NxP>
+        The <NxCode>NxBinaryDonutChart</NxCode> accepts a default height of 30px (without padding or margin), which
+        can be modified by accessing the CSS height property on the <NxCode>.nx-binary-donut-chart</NxCode> class.
+      </NxP>
+      <NxP>
+        By default the CSS property of <NxCode>NxBinaryDonutChart</NxCode> is set to a value of
+        inline. <NxCode>Display:inline</NxCode> will not generate any line breaks to maintain the flow of the
```

Comment:
Code rules take precedence over sentence rules```suggestion
        inline. <NxCode>display:inline</NxCode> will not generate any line breaks to maintain the flow of the
```

Reply:
ok, I'll make the change